### ISSUE TITLE
[BUG] Spurious changes in ./bin/utils/export_docs_generators.sh output on Windows

### ISSUE THREAD

**Issue by hauntingEcho** [OpenAPITools/openapi-generator#9401]
#### Bug Report Checklist

- [ X ] Have you provided a full/minimal spec to reproduce the issue?
- [ N/A ] Have you validated the input using an OpenAPI validator ([example](https://apidevtools.org/swagger-parser/online/))?
- [ X ] Have you [tested with the latest master](https://github.com/OpenAPITools/openapi-generator/wiki/FAQ#how-to-test-with-the-latest-master-of-openapi-generator) to confirm the issue still exists?
- [ X ] Have you searched for related issues/PRs?
- [ X ] What's the actual output vs expected output?
- [ ] [Optional] Sponsorship to speed up the bug fix or feature request ([example](https://github.com/OpenAPITools/openapi-generator/issues/6178))

##### Description

running the steps required as part of the PR template generates a bunch of junk on Windows, from filepath (/ vs \) differences

##### openapi-generator version

master branch

##### Generation Details

running the steps required as part of the PR template

##### Steps to reproduce

1. Open git-bash on Windows
2. `git checkout master && git reset --hard && git clean -fdx` to make sure you don't have any junk in your local repo
3. `./mvnw clean package`
4. `./bin/utils/export_docs_generators.sh` to generate the samples
5. `git diff` to see the unwanted changes.

Expected output: nothing in the diff
Actual output:  very long, containing a lot of spurious changes.  See attached 
[tmp.txt](https://github.com/OpenAPITools/openapi-generator/files/6423441/tmp.txt)

**Comment by wing328** [comment]
Agreed there shouldn't be unwanted changes when running the script in windows.

Looks like `./bin/utils/export_docs_generators.sh` needs to be updated to make it OS-independent.
